test/test_sqlalchemy_json.py

```diff
@@ -41,6 +41,12 @@ class Article(Base):
     references = Column(NestedMutableJson)
 
 
+class Dummy(Base):
+    __tablename__ = "dummy"
+    id = Column(Integer, primary_key=True)
+    content = Column(NestedMutableJson)
+
+
```

Comment:
I think this was accidentally committed from #44 and shouldn’t be part of this PR.